config/routes.rb

```diff
@@ -5,7 +5,14 @@
   get :sha, controller: :heartbeat
   get :reporting, controller: :reporting
 
+  # NOTE: this has questionable existance.
```

Comment:
do we actually need this?

Reply:
I don't think this is needed any more. originally we had an open api spec but no way of presenting it, this was then added to fit that space. However it needed to be gov branded so was not fit for purpose. This is now presented thru the tech docs gem